packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -707,6 +733,16 @@ contract Deploy is Deployer {
         require(addressManager.owner() == proxyAdmin);
     }
 
```

Comment:
For the contract names that are passed to `save` as part of this PR, can you update the following struct with the same names? https://github.com/ethereum-optimism/optimism/blob/b0f845c2ef298b63a4551be2937c65d671f20a9b/op-chain-ops/genesis/config.go#L649